tools/build/configure.py

```diff
@@ -13,7 +13,7 @@
 CPPFLAGS = "-w -Iver/$version/build/include -Iinclude -Isrc -Iassets/$version -D _LANGUAGE_C -D _FINALROM -D VERSION=$version " \
             "-ffreestanding -DF3DEX_GBI_2 -D_MIPS_SZLONG=32 -MD -MF $out.d"
 
-ASFLAGS = "-EB -G 0"
+ASFLAGS = "-G0"
```

Comment:
Just noticing this now but ideally we shouldn't use `-g0` (clear debug info?) when --debug is passed

Reply:
unless ofc -G does something different for as